### DISCUSSION TITLE
Prices from yahoo finance for mutual funds are zero

### DISCUSSION THREAD

**Discussion by pmgmarques** [ananthakumaran/paisa#201] [Ideas]
When getting prices of funds from Yahoo, day prices are always zero: [example](https://finance.yahoo.com/quote/0P0001BIL8.F/history)
Paisa should consider the last valid price instead of zero.
<img width="926" alt="Capturar" src="https://github.com/ananthakumaran/paisa/assets/3091257/e095c05a-f3cf-4c8a-944e-af33465aad4b">

**Comment by ananthakumaran** [comment]
does it always happen to you? I noticed it a couple of times, but if I wait for a day or so it will get fixed. Yahoo seems to have a bug I guess, I will see if I can filter out 0s

**Reply by pmgmarques** [reply]
It doesn't happen with all the quotes, only with some of the funds.
Today it shows zero in 0P0001BIL8.F
